templates/CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.tpl

```diff
@@ -38,7 +38,9 @@
 CRM.$(function($) {
 
   function checkPeriod() {
-    var speriod = $('#prior_financial_period').val();
+    var m = $('#prior_financial_period_M').val();
+    var y = $('#prior_financial_period_Y').val();
+    var speriod = m + '-' + y;
     var hperiod = '{/literal}{$priorFinancialPeriod}{literal}';
     if (((hperiod && speriod == '') || (hperiod && speriod != '')) && (speriod != hperiod)) {
```

Comment:
Is there a simpler way to write ((hperiod && speriod == '') || (hperiod && speriod != '')) : it seems to be testing that hperiod is truthy ie not (null or undefined or 0 or "" (the empty string) or false or NaN), and some odd condition for speriod - I'm not sure when (speriod=='' || speriod != '') is false.